src/test/java/com/datadog/api/ClientSteps.java

```diff
@@ -148,6 +151,23 @@ public void setBody(String data)
     world.addRequestParameter("body", data);
   }
 
+  @Given("body from file {string}")
+  public void setBodyFromFile(String filename)
+      throws java.lang.IllegalAccessException, java.lang.NoSuchFieldException,
+          java.lang.ClassNotFoundException, java.lang.NoSuchMethodException,
+          java.lang.reflect.InvocationTargetException, IOException {
+    String data =
+        String.join(
+            "\n",
+            Files.readAllLines(
+                Paths.get(
+                    "src/test/resources/com/datadog/api/"
+                        + apiVersion
+                        + "/client/api/"
+                        + filename)));
```

Comment:
This would be maybe easier to read:
```suggestion
    String bodyPath = Paths.get("src/test/resources/com/datadog/api/" + apiVersion + "/client/api/" + filename);
    String data = new String(Files.readAllBytes(bodyPath));
```

Reply:
Pre-commit formatter did that ¯\_(ツ)_/¯